nautobot/ipam/views.py

```diff
@@ -818,6 +840,14 @@ def post(self, request):
             },
         )
 
+    def assign_ipaddresses_to_interface(self, pks, request):
+        ip_addresses = IPAddress.objects.filter(pk__in=pks)
+        interface_model = Interface if "interface" in request.GET else VMInterface
+        interface_id = request.GET.get("interface") or request.GET.get("vminterface")
+        interface = interface_model.objects.get(id=interface_id)
+        interface.ip_addresses.set(ip_addresses)
```

Comment:
Should this be `.add` instead of `.set`?

Reply:
`set` since `ip_addresses` is an `list`.

Reply:
Right, but `set()` will replace any existing interface.ip_addresses won't it? `add()` would add to any existing ip_addresses instead of replacing them.